src/components/DashboardPage.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Filters from "./Filters";
+
+const DashboardPage = props => (
+  <div>
+    dashboard
```

Comment:
Here I have the same remarks as in the `AddEventPage` component with `div` and HTML tags.